libs/avro/Makefile

```diff
@@ -9,20 +9,19 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=avro-c
 PKG_VERSION:=1.8.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@APACHE/avro/avro-$(PKG_VERSION)/c
 PKG_HASH:=4639982b2b8fbd91fc7128fef672207129c959bb7900dd64b077ce4206edf10e
 
 PKG_MAINTAINER:=John Clark <inindev@gmail.com>
-
-PKG_INSTALL:=1
```

Comment:
PKG_INSTALL is meant to "do the right thing" regardless of what tooling is being used.  If that's not working, that's a bug that should be fixed in core, not in packages one by one.  https://openwrt.org/docs/guide-developer/packages makes no mention of this "CMAKE_INSTALL" and is the only documentation, so I'd consider this patch a regression into undocumented wilderness rather than any sort of fix.

Reply:
https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L3
https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L3

Reply:
So add it to the "creating packages" documentation.  this is insane, removing documented code and replacing it with magic internals "because"

Reply:
Honestly, the if CMAKE_INSTALL should be elimianted, there's no reason for the second one.  You're fixing symptoms instead of causes.